Package.swift

```diff
@@ -38,13 +38,14 @@ var package = Package(
         .target(
             name: "IBLinterKit",
             dependencies: [
-                "IBDecodable", "Commandant",
-                "SourceKittenFramework", "XcodeProjKit"
+                "IBDecodable",
+                "SourceKittenFramework", "XcodeProjKit",
+                .product(name: "ArgumentParser", package: "swift-argument-parser"),
             ]
         ),
         .target(
             name: "IBLinterTools",
-            dependencies: ["IBLinterKit", "Commandant"]
```

Comment:
I would like to leave an explicit dependency declaration for argument parser because a dependency of a dependency X should not be known by a dependent, that depends on X

Reply:
Fixed at e70cce0